extensions/wikia/AppPromoLanding/AppPromoLandingController.class.php

```diff
@@ -36,7 +36,7 @@ public function index() {
 
 		// Since this "Community_App" article won't be found, we need to manually say it's okay so that it's not a 404.
 		$this->response->setCode( self::RESPONSE_OK );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)